dashboard/db/schema.rb

```diff
@@ -2373,12 +2383,8 @@
     t.index ["word", "definition"], name: "index_vocabularies_on_word_and_definition", type: :fulltext
   end
 
-  add_foreign_key "census_submission_form_maps", "census_submissions"
```

Comment:
There's a lot of unnecessary diff here, and I'm not sure where it came from. I don't have time to troubleshoot because I'm OOO the rest of the week, but I'll look when I'm back!